LEAF_Request_Portal/sources/FormEditor.php

```diff
@@ -591,11 +591,32 @@ public function setFormType($categoryID, $input){
         return $result;
     }
 
-    public function setFormDestructionAge(string $categoryID, int $input): int|null {
-        if ($input === 0) {
-            $input = null;
+    /**
+     * Create age (days) for destruction records
+     *
+     * @param string $categoryID - category having its destructionAge set
+     * @param int|null $input - number of days to mark a record for destruction
+     *
+     * @return int|null
+     */
+    public function setFormDestructionAge(string $categoryID, int $input = null): int|null {

```

Comment:
Since this is all brand new, we should be able to set the return up right.  Meaning the array/json version with the two parts.